pkg/lvm/mount.go

```diff
@@ -199,3 +199,32 @@ func MountFilesystem(vol *apis.LVMVolume, mount *MountInfo) error {
 
 	return MountVolume(vol, mount)
 }
+
+// MountBlock mounts the block disk to the specified path
+func MountBlock(vol *apis.LVMVolume, mountinfo *MountInfo) error {
+	target := mountinfo.MountPath
+	volume := vol.Spec.VolGroup + "/" + vol.Name
+	devicePath := DevPath + volume
+
+	mountopt := []string{"bind"}
```

Comment:
Why do we need to bind mount the device at a path during nodepublish. Is it a requirement from CSI?

Reply:
yes, we need to moount the device to the path where kubelet wants us to.